in-person-payments-example/README.md

```diff
@@ -0,0 +1,147 @@
+# Adyen [In-person Payment Demo](https://docs.adyen.com/point-of-sale/) Integration Demo
```

Comment:
I think the README should have the same structure and content (unless language-specific notes) as the IPP Node README https://github.com/adyen-examples/adyen-node-online-payments/tree/main/in-person-payments-example